doc/getting_started/setup_verilator.md

```diff
@@ -30,7 +30,7 @@ git checkout v$VERILATOR_VERSION
 autoconf
 CC=gcc-11 CXX=g++-11 ./configure --prefix=/tools/verilator/$VERILATOR_VERSION
 CC=gcc-11 CXX=g++-11 make
-CC=gcc-11 CXX=g++-11 make install
+sudo CC=gcc-11 CXX=g++-11 make install
```

Comment:
I would be hesitant to put `sudo` in code that people might copy and paste, I would prefer to add a note in the text below.

Reply:
I think this is OK, it should always work on our supported system.

Reply:
Yeah, I saw that there is sudo in other places in the doc as well. So let's just keep it this way.